pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -107,6 +123,10 @@ func (p *staticPolicy) Allocate(ctx context.Context, s state.State, pod *v1.Pod,
 		return nil
 	}
 
+	if p.isPodWithPodLevelResources(ctx, pod) {
+		return MemoryManagerPodLevelResourcesError{}
+	}
```

Comment:
Similar to CPU manager - we only want to emit this if it _would_ have been memory aligned if using container resources, right?